src/components/Login.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { useHistory } from "react-router-dom";
+
+function Login() {
+  const history = useHistory();
+  const handleClick = (e) => {
+    e.preventDefault();
+    history.replace("/form");
+  };
+  const regClick = (e) => {
+    e.preventDefault();
+    history.replace("/signup");
+  };
```

Comment:
No need of `e` and `e.preventDetault`.

Just add an attribute to the button. 
```html
<button type="button"> ... </button>
```

Reply:
Done